spec/models/context_spec.rb

```diff
@@ -38,11 +38,11 @@
   end
 
   describe 'to_h' do
-    let(:user) { double 'User' }
-    let(:system_settings) { double 'SystemSettings' }
+    let(:user) { instance_double User, 'admin_role?' => true }
+    let(:system_settings) { instance_double SystemSetting }
     let(:context) { Context.new user: user, system_settings: system_settings }
 
-    def method_with_keyword_args positional_arg, non_context_keyword_arg:, user:, system_settings:
+    def method_with_keyword_args positional_arg, non_context_keyword_arg:, user:, system_settings:, **_rest
```

Comment:
Why is this needed?

Reply:
Great question! I didn't mention anything about this in the commit or PR -- it's a bit irrelevant and i forgot about it :).

This spec itself is a bit confusing IMO, and maybe not even necessary. I was experimenting with how to easily destructure `context` into method call args, for the cases where the method needs multiple context properties.

The destructuring works nicely with `.to_h`, but in order for it to be really resilient, the method has to accept a 'glob' arg to cover all the other context attributes it doesn't care about. In this case, adding the new `host_organization` attribute to `Context.to_h` broke this spec -- because `method_with_keyword_args` wasn't expecting it.